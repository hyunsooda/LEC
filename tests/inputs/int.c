#include <stdio.h>
#include <stdlib.h>

int main() {
    char SIZE = 15;
    int v1 = 7;
    int v2 = 8;
    int* intarr = (int*)malloc(sizeof(int) * SIZE);
    intarr[4] = 123;
    printf("%d\n", intarr[v1 * v2]);
    exit(1);
}
