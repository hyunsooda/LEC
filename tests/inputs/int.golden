Found out of bound access: ["int.c":10:20]: 
	 array length: 15, indexed by: 56 
 	 variable name: "intarr", allocated at: 8
